lib/common.js

```diff
@@ -16,7 +16,13 @@
 'use strict';
 
 const Path = require('path');
-const Thing = require('core-util-is');
+
+/**
+ * Returns `true` if the given object is strictly an `Object` and not a `Function` (even though functions are objects in JavaScript). Otherwise, returns `false`
+ * @param {unknown} val
+ * @returns {boolean}
+ */
+const isObject = (value) => value !== null && typeof value === "object";
```

Comment:
Might be confusing to have isObject that can be array, maybe we should just move the Array.isArry check here?

Reply:
I agree but I was leaving the functionality 1:1

https://github.com/isaacs/core-util-is/blob/master/lib/util.js#L73-L76